assets/components/directDebit/directDebitActions.js

```diff
@@ -10,7 +10,7 @@ import { checkAccount } from './helpers/ajax';
 export type Action =
   | { type: 'DIRECT_DEBIT_POP_UP_OPEN' }
   | { type: 'DIRECT_DEBIT_POP_UP_CLOSE' }
-  | { type: 'DIRECT_DEBIT_UPDATE_SORT_CODE', sortCode: string }
+  | { type: 'DIRECT_DEBIT_UPDATE_SORT_CODE', index: number, partialSortCode: string }
```

Comment:
I wonder if it is possible to restrict the index even more with: 
```js
type IndexType = 0 |1 |2;
inde: IndexType;
```

Reply:
I'll take a look